### DISCUSSION TITLE
Is PNPM support adding package from Monorepo repository?

### DISCUSSION THREAD

**Discussion by lamualfa** [pnpm/pnpm#3248] [Q&A]
I have to install a package from Github where that repo use a monorepo concept. If i execute `pnpm add -D github:vitejs/vite`, the installed package is `"vite-monorepo": "github:vitejs/vite"`, not `"vite": "^2.0.5"`. To install `vite`, i need to install it from the sub directories that located in [vitejs/vite/packages/vite](https://github.com/vitejs/vite/tree/main/packages/vite). How to install a package from sub directories of the repository?

**Comment by zkochan** [comment]
It is not possible at the moment.

**Reply by lamualfa** [reply]
Should we give a warning or a note about it?
